beeflow/common/config_driver.py

```diff
@@ -225,25 +225,45 @@ def filepath_completion_input(*pargs, **kwargs):
 DEFAULT_SCHED_PORT = 5100 + OFFSET
 
 DEFAULT_BEE_WORKDIR = join_path(HOME_DIR, '.beeflow')
+DEFAULT_BEE_DROPPOINT = join_path(HOME_DIR, '.beeflow/droppoint')
 USER = getpass.getuser()
 # Create the validator
 VALIDATOR = ConfigValidator('BEE configuration file and validation information.')
 VALIDATOR.section('DEFAULT', info='Default bee.conf configuration section.')
+
 VALIDATOR.option('DEFAULT', 'bee_workdir', info='main BEE workdir',
                  default=DEFAULT_BEE_WORKDIR, validator=validation.make_dir)
+
+VALIDATOR.option('DEFAULT', 'bee_droppoint', info='BEE remote workflow drop point',
+                 default=DEFAULT_BEE_DROPPOINT, validator=validation.make_dir)
+
+VALIDATOR.option('DEFAULT', 'remote_api', info='BEE remote REST API activation',
+                 default=False, validator=validation.bool_)
+
+VALIDATOR.option('DEFAULT', 'remote_api_port', info='BEE remote REST API port',
+                 default=7777, validator=int)
+
 VALIDATOR.option('DEFAULT', 'workload_scheduler', choices=('Slurm', 'LSF', 'Flux', 'Simple'),
                  info='backend workload scheduler to interact with ')
+
+VALIDATOR.option('DEFAULT', 'use_archive', validator=validation.bool_, default=True,
```

Comment:
We aren't using use_archive and got rid of all instances. We don't want it back.

Reply:
Sounds good. I got rid of that code.